### ISSUE TITLE
Navbar Text Color Contrast is Too Low for Accessibility

### ISSUE THREAD

**Issue by YOGINIMAHIMA1** [GarimaSingh0109/WasteManagment#155]
The text color of the navigation bar (e.g., "Upload," "Features," "Feedback," "Signup," "Contact") appears to have insufficient contrast against the background color.

**Comment by wouffle** [comment]
Please assign this to me @GarimaSingh0109
